draft-jennings-moq-secure-objects.md

```diff
@@ -193,28 +193,17 @@ qualified domain names or UUIDs as part of the TrackNamespace.
 
 # Secure Objects
 
-A Secure MOQT Object is MOQT OBJECT_STREAM or OBJECT_DATAGRAM messages that has
-been protected with the scheme defined in this draft, so that its payload is
-encrypted and the whole message is authenticated by an authentication tag in the
-payload:
+{{Section 7.1.1 of I-D.ietf-moq-transport}} defines fields of a canonical
+MOQT Object. The protection scheme defined in this draft encrypts the
+`Object Payload` and the fields `Track Alias`, `Group ID`, `Object ID`,
+`Object Payload` are authenticated, regardless of the on-the-wire encoding
+of the objects over QUIC Datagrams or QUIC streams.
 
-~~~ aasvg
-OBJECT_STREAM Message {
-  Subscribe ID (i),
-  Track Alias (i),
-  Group ID (i),             <-- authenticated
-  Object ID (i),            <-- authenticated
-  Object Send Order (i),
-  Object Status (i),
-  Object Payload (..),      <-- encrypted and authenticated
-}
-~~~
-{: #fig-encrypted-object title="Security properties of a secure OBJECT_STREAM
-object.  Protection of an OBJECT_DATAGRAM message is analogous." }
+TODO: Move authentication tag to Object Header Extension.
```

Comment:
```suggestion
```

No, this should not be done.  According to the AEAD interface in [RFC 5116](https://datatracker.ietf.org/doc/html/rfc5116#section-2.1), there is no separate authentication tag, just a unitary ciphertext.